ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -73,6 +92,24 @@ class CellEditorOverlay extends Component {
     }
   }
 
+  handleSetRange(e) {
+    const {min, max} = e.target.form
+    const {axes} = this.state
+
+    this.setState({
+      axes: {...axes, y: {...axes.y, bounds: [min.value, max.value]}},
+    })
+    e.preventDefault()
+  }
+
+  handleSetLabel(e) {
+    const {label} = e.target.form
```

Comment:
These patterns are almost identical to the first example of controlled components in the [React docs for forms](https://facebook.github.io/react/docs/forms.html)